Lib/platform.py

```diff
@@ -398,9 +398,9 @@ def win32_ver(release='', version='', csd='', ptype=''):
     else:
         try:
             cvkey = r'SOFTWARE\Microsoft\Windows NT\CurrentVersion'
-            with winreg.OpenKeyEx(HKEY_LOCAL_MACHINE, cvkey) as key:
-                ptype = QueryValueEx(key, 'CurrentType')[0]
-        except:
+            with winreg.OpenKeyEx(winreg.HKEY_LOCAL_MACHINE, cvkey) as key:
+                ptype = winreg.QueryValueEx(key, 'CurrentType')[0]
+        except OSError:
```

Comment:
Is there any reason for reducing exception scope?

Reply:
IMHO, if you want to update exception handling.
please do it for seperate issue.

Reply:
Isn't this what was allowing the NameError to pass silently? Is there a reason for this line to be different than line 356, or is it just a matter of being extra-careful not to raise any new exceptions?

Reply:
@sweeneyde 
This change can change another exception handling which can cause regression issues. 
So if you want to handle this issue. Please create a new issue on bugs.python.org